syntax = "proto3";
import "google/protobuf/empty.proto";
package directors;

message Director {
  int32 id = 1;
  string name = 2;
  string surname = 3;
}

service DirectorsService {
  rpc get_directors (google.protobuf.Empty) returns (stream Director);
  rpc get_director (DirectorId) returns (Director);
}

message DirectorId {
  int32 id = 1;
}